builder-automate.md

```diff
@@ -0,0 +1,270 @@
+
+1. For existing Chef Automate installations, move the `cert.pem` that you saved /hab/cache/ssl
+### Step Four: Install Builder
+
+1. Run the install script. This installs both Chef Habitat Builder on-prem and the Chef Habitat datastore:
+
+    ```bash
+    bash ./install.sh
+    ```
+
+1. Accept both licenses.
+1. All services should report back as `up`. It make take a few minutes to come up.
+
+    ```bash
+    sudo hab svc status
+    ```
+
+    Should return something similar to:
+
+    ```output
+    package                                        type        desired  state  elapsed (s)  pid    group
+    habitat/builder-api/8473/20190830141422        standalone  up       up     595          28302  builder-api.default
+    habitat/builder-api-proxy/8467/20190829194024  standalone  up       up     597          28233  builder-api-proxy.default
+    habitat/builder-memcached/7728/20180929144821  standalone  up       up     597          28244  builder-memcached.default
+    habitat/builder-datastore/7809/20181019215440  standalone  up       up     597          28262  builder-datastore.default
+    habitat/builder-minio/7764/20181006010221      standalone  up       up     597          28277  builder-minio.default
+    ```
+
+### Step Five: Copy Automate's Certificate to the `/hab/cache/ssl` Directory
+
+1. View and copy the Chef Automate certificate. Change the server name to your Chef Automate installation FQDN:
+
+    ```bash
+    openssl s_client -showcerts -servername chef-automate.test -connect chef-automate.test:443 < /dev/null | openssl x509
+    ```
+
+    Copy the output to an accessible file.
+
+    ```output
+    # Copy the contents including the begin and end certificate
+    # -----BEGIN CERTIFICATE-----
+    # Certificate content here
+    #-----END CERTIFICATE-----
+    ```
+
+1. Make a file for you cert at `/hab/cache/ssl/`, such as `automate-cert.crt`. For a `.pem` file, `automate-cert.pem`. Overwriting `cert.pem` will cause your Builder installation to fail.
+1. Paste the Chef Automate certificate into your file, `/hab/cache/ssl/automate-cert.crt`
+1. For existing Chef Automate installations, copy the `automate-cert.pem` file that you saved on your workstation in step one to the `/hab/cache/ssl/` directory.
+1. Restart builder
+
+    ```bash
+    sudo systemctl restart hab-sup
+    ```
+
+### You're Done!
+
+1. Login at
+
+    ```bash
+    https://chef-builder.test
+    ```
+
+## Errors
+
+### Unable to retrieve OAuth token
+
+You were able to sign in to Chef Automate, but Chef Automate was unable to authenticate Chef Habitat's OAuth token.
+
+Open the `bldr.env` and verify that:
+
+* **APP_URL** ends with "/\"
+* **OAUTH_REDIRECT_URL** ends with "/\"
+* **OAUTH_CLIENT_ID** is complete and correct
+* **OAUTH_CLIENT_SECRET** is complete and correct
+
+Apply changes to the to apply changes to the `bldr.env`
+
+```bash
+bash ./install.sh
+```
+
+Restart the Chef Habitat services:
+
+```bash
+sudo systemctl restart hab-sup
+```
```

Comment:
Hi @kagarmoe! I'm not completely certain why we're adding this document in my PR?  While I seem to agree with the file's contents, it feels a little out of scope from the intent of my PR.

I just don't want it to get in the way of the team's approval. If the team approves of this, I'm fine to leave this in.
Thank you!

Reply:
Sorry, an unaffiliated file got pushed when I corrected a few minor markdown formatting issues.

Reply:
No worries. Thanks @kagarmoe!